### PR TITLE
Weekly merge

### PR DESCRIPTION
**Pull request by orinevares**
- Link, textarea, date input, and table added to components
- Accessibility fixes to checkbox and radio button code
- Accessibility fixes to rest of code - english lang attribute, alt text to logos etc
- Various MD file edits

### CODE REVIEW

File: components/checkbox/style.css [hunk 1]
```diff
@@ -37,6 +37,11 @@
   background-color: #003366;
 }
 
+/* Custom checkbox has blue outline when in focus */
+.checkbox input:focus ~ .checkmark {
+  outline: rgb(59, 153, 252) solid 5px;
```
Comment (jleach) [inline components/checkbox/style.css]: I think we're using hex values in other CSS. Might be worth using them here for consistency.

File: components/date_input/README.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+The library is constantly updated and currently available in the following tools:
+
+*	[Sketch](https://sketch.cloud/s/Q0bkG)
+* [UxPin and others]
+
+## Code
+
+### HTML
+```
```
Comment (jleach) [inline components/date_input/README.md]: You can often get syntax highlighting by adding the code type right after the third back-tick. For example add `html` or `css`. 
```html
<h1> Hello </h1>
```

File: components/date_input/README.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
+---
+description: Date input allows users to enter an already known date
+title: Date Input
+status: Draft
+---
+
```
Comment (jleach) [inline components/date_input/README.md]: Does this one need the status badge?

File: components/table/README.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
+---
+description: Tables present tabular data in a grid which allows users to visually scan and make visual associations between the data.
+title: Table
+status: Draft
+---
+
```
Comment (jleach) [inline components/table/README.md]: Status badge?

File: components/date_input/README.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+## Requirements
+* This component must be used according the the standard outined in this document. Any feedback should be directed to the feedback forum.
+
+## How to Use This
+*	Use when users need to enter an easily memorable date
+
+## Don't Use This For
+*	Dates that cannot be easily remembered
+*	When users have to schedule something in the future, a calendar dropdown may be better.
```
Comment (bl-smith) [inline components/date_input/README.md]: I wonder if we should recommend a calendar dropdown here, as it isn't something we've tested or researched. Could we just say "another option may be better", and wait until we have a more specific recommendation. I'm concerned that a calendar dropdown may not actually be the right or accessible choice

File: components/textarea/README.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
+---
+description: Textarea allow users to input multiple lines of text
+title: Textarea
+status: Draft
+---
+
```
Comment (jleach) [inline components/textarea/README.md]: Status badge?

File: components/date_input/README.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+## Code
+
+### HTML
+```
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
```
Comment (orinevares) [inline components/date_input/README.md]: Change to noto sans

File: components/date_input/README.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+    </form>
+
+  </body>
+</html>
+```
+  
+### CSS
+```
+form {
+  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
```
Comment (orinevares) [inline components/date_input/README.md]: change to noto sans

File: components/date_input/README.md [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+}
+
+.date_input_day, .date_input_month {
+  height: 25px;
+  border: 2px solid #38598a;
+  margin-top: 5px;
+  margin-bottom: 15px;
+  border-radius: 3px;
+  padding: 5px;
+  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
```
Comment (orinevares) [inline components/date_input/README.md]: noto sans

File: components/date_input/README.md [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+}
+
+.date_input_year {
+  height: 25px;
+  border: 2px solid #38598a;
+  margin-top: 5px;
+  margin-bottom: 15px;
+  border-radius: 3px;
+  padding: 5px;
+  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
```
Comment (orinevares) [inline components/date_input/README.md]: noto sans

File: components/date_input/sample.html [hunk 11]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
```
Comment (orinevares) [inline components/date_input/sample.html]: noto sans

File: components/date_input/style.css [hunk 12]
```diff
@@ -0,0 +1,52 @@
+form {
+  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
```
Comment (orinevares) [inline components/date_input/style.css]: noto sans

File: components/date_input/style.css [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.date_input_day, .date_input_month {
+  height: 25px;
+  border: 2px solid #38598a;
+  margin-top: 5px;
+  margin-bottom: 15px;
+  border-radius: 3px;
+  padding: 5px;
+  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
```
Comment (orinevares) [inline components/date_input/style.css]: noto sans

File: components/date_input/style.css [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.date_input_year {
+  height: 25px;
+  border: 2px solid #38598a;
+  margin-top: 5px;
+  margin-bottom: 15px;
+  border-radius: 3px;
+  padding: 5px;
+  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
```
Comment (orinevares) [inline components/date_input/style.css]: noto sans

File: components/table/sample.html [hunk 15]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
```
Comment (orinevares) [inline components/table/sample.html]: noto sans

File: components/textarea/README.md [hunk 16]
```diff
@@ -0,0 +1,127 @@
...
+## Code
+
+### HTML
+```
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
```
Comment (orinevares) [inline components/textarea/README.md]: noto sans

File: components/textarea/README.md [hunk 17]
```diff
@@ -0,0 +1,127 @@
...
+```  
+### CSS
+```
+form {
+  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
+  font-size: 16px;
+}
+
+.text_input {
+  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
```
Comment (orinevares) [inline components/textarea/README.md]: noto sans

File: components/textarea/sample.html [hunk 18]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans" rel="stylesheet">
```
Comment (orinevares) [inline components/textarea/sample.html]: noto sans

File: components/textarea/style.css [hunk 19]
```diff
@@ -0,0 +1,24 @@
+form {
+  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
```
Comment (orinevares) [inline components/textarea/style.css]: noto sans

File: components/textarea/style.css [hunk 20]
```diff
@@ -0,0 +1,24 @@
...
+  font-size: 16px;
+}
+
+.text_input {
+  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
```
Comment (orinevares) [inline components/textarea/style.css]: noto sans

File: components/date_input/README.md (hunk 2 above)
Comment (orinevares) [reply components/date_input/README.md]: Awesome, thanks for the tip!

File: components/textarea/README.md [hunk 21]
```diff
@@ -0,0 +1,129 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
+    <title>Textarea</title>
+  </head>
+  <body>
+
+    <form>
+      <div class="text_label">
+        <label>Can you provide more detail?</label>
+      </div>
+      <textarea class="text_input" name="name" rows="8" cols="80"></textarea>
```
Comment (bl-smith) [inline components/textarea/README.md]: Is this code mobile responsive? Will it resize with the screen size?

File: components/link/README.md [hunk 22]
```diff
@@ -0,0 +1,103 @@
...
+* Users can identify the link without relying on colour alone
+
+### Tab Navigation
+* Supports tab navigation for use without a mouse
+
+### Learn More
+* Contacts
+* Online Tools
+* Online Resources
+* Gov Resources
```
Comment (bl-smith) [inline components/link/README.md]: If these Learn More resources end up being links, we should make sure that we follow our own advice about how to write links descriptively

File: components/link/README.md [hunk 23]
```diff
@@ -0,0 +1,103 @@
...
+## Don't Use This when:
+* Users need to take an action on a page such as log-in or submit, use a primary button instead.
+
+## Design Protocol
+* If your link is at the end of a paragraph or sentence make sure the linked text does not include the full stop.
+
+## Behaviour
+
+1.	Link starts out as government link colour, #1a5a96.
+2.	On hover link removes underline and turns to blue.
```
[the remark this answers is not in the document]
Comment (orinevares) [reply components/link/README.md]: Hmm good idea

**Pull request merged**